vendor/github.com/f5devcentral/go-bigip/ltm.go

```diff
@@ -501,6 +501,7 @@ type VirtualServer struct {
 	ConnectionLimit            int    `json:"connectionLimit,omitempty"`
 	Destination                string `json:"destination,omitempty"`
 	Enabled                    bool   `json:"enabled,omitempty"`
+	Disabled                   bool   `json:"disabled,omitempty"`
```

Comment:
ok, so this seems to relate to my previous comment and I don't quite understand why we need two properties to specify a single state?

isn't `Enabled (bool) == false` already means **Disabled**? why do we need a second property?

how would the api call work when both `Enabled` and `Disabled` are set to `true`?

Reply:
Above comment explains this requirement